### DISCUSSION TITLE
Shake-to-undo doesn’t work [ios]

### DISCUSSION THREAD

**Discussion by jasonkarns** [community/community#128283] [Mobile] [answered]
iOS convention is for shake to trigger undo.

After archiving a notification inadvertently, I shook. Nothing undid.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.165.0 (133064262)
Device: iPhone 13 mini (iOS 17.5.1)
</details>

**Comment by elisealix22** [comment, marked as answer]
Hi @jasonkarns , thanks for reaching out! I opened an issue to track this internally and will surface it with our team. Please let us know if you have any other suggestions in the meantime!

**Reply by jasonkarns** [reply]
Still looking for this. Another notification was marked "done" accidentally and there is no way to undo that. Again I instinctively used shake gesture and nothing happened. Unable to restore that notification.
